src/Components/pages/Play.jsx

```diff
@@ -27,24 +24,34 @@ class Play extends Component {
     constructor (props) {
         super(props);
         this.state = {
-            game: {},
-            gameId: null,
+            //static properties
+            roomId: null,
             password: '',
-            phase: 'not-joined',
-            nickname: '',
             userId: '',
+            nickname: '',
+            numSymbols: 0,
+
+            //frequently updated properties
+            phase: 'not-joined',
             questionNum: 0,
-            chartURLs: [],
-            technicalIndicators: [],
-            currentCash: 0,
-            currentShares: [],
-            currentPrice: null,
+            chartURLs: null,
+            net_worth: 0,
+            money_left: [],
+            curShares: [],
```

Comment:
why are these being tracked in play? aren't these user-specific?

Reply:
yes, play is user specific after it authenticates the user